{
  "id": "oval:com.redhat.rhsa:def:20171470",
  "version": "636",
  "class": "patch",
  "metadata": {
    "title": "RHSA-2017:1470: python-django security update (Moderate)",
    "reference": [
      {
        "ref_id": "RHSA-2017:1470",
        "ref_url": "https://access.redhat.com/errata/RHSA-2017:1470",
        "source": "RHSA"
      },
      {
        "ref_id": "CVE-2017-7233",
        "ref_url": "https://access.redhat.com/security/cve/CVE-2017-7233",
        "source": "CVE"
      }
    ],
    "description": "Django is a high-level Python Web framework that encourages rapid development and a clean, pragmatic design. It focuses on automating as much as possible and adhering to the DRY (Don't Repeat Yourself) principle.\n\nSecurity Fix(es):\n\n* A redirect flaw, where the is_safe_url() function did not correctly sanitize numeric-URL user input, was found in python-django. A remote attacker could exploit this flaw to perform XSS attacks against the OpenStack dashboard. (CVE-2017-7233)\n\nRed Hat would like to thank the Django project for reporting this issue.",
    "advisory": {
      "from": "secalert@redhat.com",
      "severity": "Moderate",
      "updated": {
        "date": "2017-06-14"
      },
      "cve": [
        {
          "text": "CVE-2017-7233",
          "cvss3": "6.1/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
          "cwe": "CWE-79",
          "href": "https://access.redhat.com/security/cve/CVE-2017-7233",
          "impact": "moderate",
          "public": "20170404"
        }
      ],
      "affected": {},
      "affected_cpe_list": {
        "cpe": [
          "cpe:/a:redhat:openstack-director:8",
          "cpe:/a:redhat:openstack-director:8::el7",
          "cpe:/a:redhat:openstack-optools:8",
          "cpe:/a:redhat:openstack-optools:8::el7",
          "cpe:/a:redhat:openstack:8",
          "cpe:/a:redhat:openstack:8::el7"
        ]
      },
      "rights": "Copyright 2017 Red Hat, Inc.",
      "issued": {
        "date": "2017-06-14"
      },
      "bugzilla": [
        {
          "text": "CVE-2017-7233 python-django: Open redirect and possible XSS attack via user-supplied numeric redirect URLs",
          "href": "https://bugzilla.redhat.com/1437234",
          "id": "1437234"
        }
      ]
    },
    "affected": {
      "family": "unix",
      "platform": [
        "Red Hat OpenStack Platform 8.0 (Liberty)"
      ]
    }
  },
  "criteria": {
    "operator": "OR",
    "criterias": [
      {
        "operator": "AND",
        "criterias": [
          {
            "operator": "OR",
            "criterias": [
              {
                "operator": "AND",
                "criterions": [
                  {
                    "test_ref": "oval:com.redhat.rhsa:tst:20171470001",
                    "comment": "python-django is earlier than 0:1.8.18-1.el7ost"
                  },
                  {
                    "test_ref": "oval:com.redhat.rhsa:tst:20161594002",
                    "comment": "python-django is signed with Red Hat redhatrelease2 key"
                  }
                ]
              },
              {
                "operator": "AND",
                "criterions": [
                  {
                    "test_ref": "oval:com.redhat.rhsa:tst:20171470003",
                    "comment": "python-django-bash-completion is earlier than 0:1.8.18-1.el7ost"
                  },
                  {
                    "test_ref": "oval:com.redhat.rhsa:tst:20161594004",
                    "comment": "python-django-bash-completion is signed with Red Hat redhatrelease2 key"
                  }
                ]
              }
            ]
          }
        ],
        "criterions": [
          {
            "test_ref": "oval:com.redhat.rhsa:tst:20161002013",
            "comment": "Red Hat Enterprise Linux 7 is installed"
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:com.redhat.rhsa:tst:20161002014",
        "comment": "Red Hat Enterprise Linux must be installed"
      }
    ]
  }
}
